### DISCUSSION TITLE
Range map dashboard

### DISCUSSION THREAD

**Discussion by stauffenberg2020** [jheredianet/Teslamate-CustomGrafanaDashboards#87] [General]
Not sure, if I am asking too much, But is it possible to make a dashboard to show the range map like what Tessie shows?

https://www.reddit.com/r/TeslaModel3/comments/zd4rp3/range_visualization_now_available_in_the_tessie/

![image](https://github.com/jheredianet/Teslamate-CustomGrafanaDashboards/assets/62483558/7b8e6db0-138b-4d71-a19d-9f844fb2b600)

**Comment by jheredianet** [comment]
Sounds interesting. I will try to research if Grafana has a similar map to plot a range.
